app/containers/EpisodeAdminEditor/index.js

```diff
@@ -12,19 +12,57 @@ import {
   selectShowsLoading,
   selectShowsError,
 } from 'containers/Shows/selectors';
+
+import {
+  selectDigasOnDemandEpisodes,
+  selectDigasPodcastEpisodes,
+  selectDigasEpisodesLoading,
+  selectDigasEpisodesError,
+} from 'containers/EpisodeAdmin/selectors';
+
+
+import {
+  selectShow,
+  selectShowEpisodes,
+  selectShowLoading,
+  selectShowError,
+} from 'containers/Show/selectors';
+
 import styles from './styles.css';
 
+
+import { updateEpisodePending, deleteEpisodePending } from './actions';
 import { loadShows } from 'containers/Shows/actions';
+import {
+  loadShowById,
+  clearShow,
+} from 'containers/Show/actions';
+import {
+  loadDigasEpisodesPending,
+  clearDigasEpisodes,
+} from 'containers/EpisodeAdmin/actions';
 
 import EpisodeForm from 'components/EpisodeForm';
 import SelectInput from 'components/SelectInput';
+import EpisodePreview from 'components/EpisodePreview';
+import DeleteButton from 'components/DeleteButton';
 
 // FieldChangeHandlerFactory
 const getFieldChangeHandler = (name) => function (event) { // eslint-disable-line func-names
   event.preventDefault();
   this.setState({ [name]: event.target.value });
 };
 
+const getDigasEpisodeHandler = (name) => function (event, episodes) { // eslint-disable-line func-names
```

Comment:
The number of eslint-disables suggests you guys should take a look at your eslintconfig =P